api/src/lang/machine.rs

```diff
@@ -122,8 +133,8 @@ impl MachineState {
 #[derive(Debug)]
 pub struct Machine {
     // Static data
-    env: Environment,
     program: Vec<MachineInstr>,
+    expected_output: Vec<LangValue>,
```

Comment:
I removed the `env` from here, because I was running into more lifetime issues when a `ProgramWebsocket` tried to hold a `Machine` and the `Environment` itself.